src/Test.UnitTests.Sarif.WorkItems/SarifWorkItemFilerTests.cs

```diff
@@ -111,13 +129,16 @@ private void TestWorkItemFiler(SarifLog sarifLog, SarifWorkItemContext context)
             //       and retrieval of the ADO filing client. We are required to put this
             //       mock behind an interface due to an inability 
 
-            var vssConnectionMock = new Mock<IVssConnection>();
-            vssConnectionMock
-                .Setup(x => x.ConnectAsync(It.IsAny<Uri>(), It.IsAny<string>()))
-                .Returns(Task.CompletedTask)
-                .Callback<Uri, string>(
-                    (uri, pat) =>
-                    {
+            FilingClient filingClient;
+            if (adoClient == true)
```

Comment:
>adoClient == true [](start = 16, length = 17)

Style nit, don't compare to bollean literal, just is if (blah). While I'm here, please factor the branches of this conditional into two distinct helpers, for best readability and factoring. Nice work on this change!